ssh/DOCS.md

```diff
@@ -76,7 +76,9 @@ Specifies whether TCP forwarding is permitted or not.
 
 ## Network
 
-This section is only relevant if you want to connect to Home Assistant using an SSH client, such as PuTTY. To enable SSH access via the network, you need to specify which port to use on the Home Assistant host. The number you enter will be used to map that port from the host into the running "Terminal & SSH" add-on. The standard port used for the SSH protocol is `22`.
+This section is only relevant if you want to connect to Home Assistant using an SSH client, such as PuTTY or Linux terminal. To enable SSH remote access from the Network, specify the desired SSH TCP server port in the Network configuration input box. The number you enter will be used to map that port from the host into the running "Terminal & SSH" add-on. The standard port used for the SSH protocol is `22`.  
+
+SSH remote access can be disabled by clearing the input box, saving the config and restarting the add-on.  The default is disabled.
```

Comment:
Small tweak to clarify it is turning off again. The section above already tells one how to enable it.

```suggestion
Remote SSH access can be disabled again, by clearing the input box, saving the configuration and restarting the add-on.
```